src/commands/app/init.js

```diff
@@ -28,9 +34,30 @@ class InitCommand extends BaseCommand {
 
     aioLogger.debug('creating new app with init command ', flags)
 
+    // default project name and services
     let projectName = path.basename(process.cwd())
     let services = 'AdobeTargetSDK,AdobeAnalyticsSDK,CampaignSDK,McDataServicesSdk,AudienceManagerCustomerSDK' // todo fetch those from console when no --import
```

Comment:
We don't need the hardcoded list anymore right? if yes at least let's remove the todo comment :)

Reply:
the services is needed to bootstrap the `gen-app` generator, no? It's one of the options.

Reply:
... when `--import` is not used (or no console generator)

Reply:
> 2 inline comments + the following:
> 
> * should we disallow --yes when no import ? cause then we would create a project without talking to console right? also in that case we won't need this logic https://github.com/adobe/aio-cli-plugin-app/blob/console-generator/src/commands/app/init.js#L93

if the console flow fails somehow for the dev, we still want them to be able to build a local app and try it out. 

I'm not familiar with the inner workings of the app plugin -- is `services` config key required somewhere?

Reply:
> if the console flow fails somehow for the dev, we still want them to be able to build a local app and try it out.

ok yes this makes a lot of sense, I thought initially we would only build an app based on the console config, but providing a fallback option is good.

> I'm not familiar with the inner workings of the app plugin -- is services config key required somewhere?

Yes let's keep it in that case, it used by the `app add action` command actually: https://github.com/adobe/aio-cli-plugin-app/blob/master/src/commands/app/add/action.js#L25. (Thought that if we only allow apps based on a console config we would then use the config services  key provided by console.)